tests/CMakeLists.txt

```diff
@@ -231,7 +231,7 @@ if(AVIF_BUILD_APPS)
             set_property(TEST test_cmd_metadata PROPERTY DISABLED True)
         endif()
 
-        # Only aom encoder supports progressive encoding.
+        # Only aom encoder supports AV1 progressive encoding.
```

Comment:
Nit: It is AVIF, not AV1, that has progressive encoding. This can be rephrased as "Only aom encoder supports encoding AV1 spatial layers.", assuming the reader knows spatial layers are used to implement AVIF layered imaged that can be progressively decoded.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1390